### PR TITLE
Adds slaves to Amazonie and Lotharingia + reworks pops

### PR DESCRIPTION
**Pull request by Glloyd**
Adds slaves to AMA and LOT, and also makes their pops more in line with DoDFF values. Also makes some Amazonie changes to reflect the DoDFF starting situation better.

### CODE REVIEW

**Comment by MadJamez** [comment]
Maybe merge main back in your branch? Seeing a bunch of errors, I think most of them are fixed on main, not sure some are caused by your changes.

**Comment by Glloyd** [comment]
will do, I forgot to checkout before getting started on this. Those errors shouldn't be from this, but I'll merge just to be safe.

**Comment by Glloyd** [comment]
should be good

File: common/history/buildings/07_south_america.txt [hunk 1]
```diff
@@ -1181,19 +1181,30 @@
 				level=1
 				reserves=1
 				activate_production_methods={ "pm_basic_shipbuilding" "pm_merchant_guilds_building_shipyards" "pm_military_shipbuilding_wooden" }
+			}
+				building="building_construction_sector"
```
Comment (MadJamez) [inline common/history/buildings/07_south_america.txt:1185]: Something is wrong here, causes a bunch of errors with buildings in south america.

Error: untyped effect [ Expected opening bracket ]
  Script location: file: common/history/buildings/07_south_america.txt line: 1498

**Review by MadJamez — CHANGES_REQUESTED** [review]

File: common/history/buildings/07_south_america.txt (hunk 1 above)
Comment (Glloyd) [reply common/history/buildings/07_south_america.txt:1186]: Yup, looks like I deleted the create_building={ when moving things around. Fixed it.

File: common/country_definitions/dvg_ameriga.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-﻿TWT = { #Tawantinsuyu
```
Comment (MadJamez) [inline common/country_definitions/dvg_ameriga.txt:1]: File encoding issue:
File 'common/country_definitions/dvg_ameriga.txt' should be in utf8-bom encoding (will try to use it anyways)
File 'common/history/buildings/07_south_america.txt' should be in utf8-bom encoding (will try to use it anyways)

File: common/history/buildings/07_south_america.txt [hunk 3]
```diff
@@ -1,4 +1,4 @@
-﻿BUILDINGS={
```
Comment (MadJamez) [inline common/history/buildings/07_south_america.txt:1]: TWT has -182 bureaucracy. Is there a reason for this?

**Review by MadJamez — CHANGES_REQUESTED** [review]

File: common/country_definitions/dvg_ameriga.txt (hunk 2 above)
Comment (Glloyd) [reply common/country_definitions/dvg_ameriga.txt]: this is what happens when I don't work on my main computer. It's clear now though.

File: common/history/buildings/07_south_america.txt (hunk 3 above)
Comment (Glloyd) [reply common/history/buildings/07_south_america.txt:1]: It's been like that for a while, and is semi-intended. Once I get working on TWT again I'll formalize it in some negative bureaucracy modifiers they'll have in the early game representing use of the quipu, which were knotted strings used in place of a writing system.

**Review by MadJamez — APPROVED** [review]

**Pull request merged**